toolchain/driver/driver.cpp

```diff
@@ -163,6 +165,22 @@ auto Driver::RunDumpSubcommand(DiagnosticConsumer& consumer,
     args = args.drop_front();
   }
 
+  llvm::StringRef output_file;
+  if (dump_mode == DumpMode::ObjectCode) {
+    if (!args.empty() && args.front().starts_with("--target_triple=")) {
```

Comment:
```suggestion
    if (!args.empty() && args.front().starts_with("--target=")) {
```
I think we can use a shorter flag name here. (Calling these "triple"s is a somewhat-LLVM-specific anachronism these days, so I'd prefer that we don't use that word in our UI.)

Reply:
Hm, looks like we're already using this flag name for `Assembly` mode, and we should be consistent. You could change them both in this PR, or keep the `--target_triple` name for now and switch to `--target` in a separate PR.

Reply:
Thank you for the suggestion! I will change it to` target` in a separate PR.